ts/store/reducers/backendStatus/__tests__/remoteConfig.test.ts

```diff
@@ -813,3 +814,69 @@ describe("isAARRemoteEnabled", () => {
     })
   );
 });
+
+describe("sendAARDelegateUrlSelector", () => {
+  const delegateUrl = "https://delegate.it";
+  const fallbackSendAARDelegateUrl =
+    "https://assistenza.notifichedigitali.it/hc/it/articles/32453819931537-Delegare-qualcuno-a-visualizzare-le-tue-notifiche";
+
+  const someState = {
+    remoteConfig: O.some({
+      pn: {
+        aar: {
+          delegate_url: delegateUrl
+        }
+      }
+    })
+  } as GlobalState;
+
+  const emptyObjectState = {
+    remoteConfig: O.some({
+      pn: {
+        aar: {}
+      }
+    })
+  } as GlobalState;
+
+  const noneState = {
+    remoteConfig: O.none
+  } as GlobalState;
+
+  const emptyStringState = {
+    remoteConfig: O.some({
+      pn: {
+        aar: {
+          delegate_url: ""
+        }
+      }
+    })
+  } as GlobalState;
+
+  const testCases = [
+    {
+      result: delegateUrl,
+      input: someState
+    },
+    {
+      result: fallbackSendAARDelegateUrl,
+      input: emptyObjectState
+    },
+    {
+      result: "",
+      input: emptyStringState
+    },
+    {
+      result: fallbackSendAARDelegateUrl,
+      input: noneState
+    }
+  ];
+
+  for (const { result, input } of testCases) {
+    it(`should return the correct result for input remoteConfig : ${JSON.stringify(
```

Comment:
When the test fails, it is hard for the reader to know which is the correct result. Can we include it in the test's it description, instead of writing a generic "correct result" ?